app/forms/state_file/id_disability_form.rb

```diff
@@ -8,19 +8,13 @@ class IdDisabilityForm < QuestionsForm
 
     def save
       if intake.filing_status_mfj?
-        primary_eligible = eligible?(:primary)
-        spouse_eligible = eligible?(:spouse)
-
         case mfj_disability
         when "me"
-          @intake.update(primary_disabled: primary_eligible ? "yes" : "no", spouse_disabled: "no")
+          @intake.update(primary_disabled: "yes", spouse_disabled: "no")
```

Comment:
[dust] the dustiest of dust, but we renamed this to `primary` for maryland for clarity & since it's not actually "me" but that doesn't super matter.

Reply:
ooh i'm going to try to change it